auth/src/main/java/org/openedx/auth/presentation/sso/OAuthHelper.kt

```diff
@@ -29,4 +29,9 @@ class OAuthHelper(
     fun clear() {
         facebookAuthHelper.clear()
     }
+
+    companion object {
+        const val ACCESS_TOKEN_EMPTY_MESSAGE = "Social Auth accessToken is empty"
+        const val ACTIVITY_CANCELLED_MESSAGE = "activity is cancelled by the user."
```

Comment:
Please capitalize the first word since it's a message. It would be okay in case of an error code.

Reply:
I just copied the google auth error response message.